src/app/routes/utils/fetchPageData/index.js

```diff
@@ -35,10 +36,14 @@ export const getUrl = pathname => {
 /**
  * An isomorphic fetch wrapper for pages, with error and log handling.
  * @param {string} path The URL of a resource to fetch.
+ * @param {number} timeout Optional parameter to provide a custom timeout
+ * for request for 'secondary data'. The fetch timeout defaults to the 'primary
+ * data' timeout if this is not provided.
```

Comment:
Is it worth providing a link to the value of the primary data timeout here?

Reply:
I've added the link in this commit: https://github.com/bbc/simorgh/pull/8120/commits/450dd8572b5194ff708a5dc4dd6f8ed2906e7691
The link won't work until we merge to latest however